ignite/distributed/launcher.py

```diff
@@ -214,7 +214,14 @@ def __init__(
             self.logger.info("- Parameters to spawn processes: \n\t{}".format(msg))
 
     @staticmethod
-    def _setup_spawn_params(nproc_per_node, nnodes, node_rank, master_addr, master_port, **spawn_kwargs):
+    def _setup_spawn_params(
+        nproc_per_node: int,
+        nnodes: Optional[int],
```

Comment:
Actually, I'd say all these args are not `Optional`, right ?

Reply:
it is due to the `__init__` function, they are all `None` by default and therefore all can be `None` except `nproc_per_node`
https://github.com/pytorch/ignite/blob/51067381b457c098b8dd80972ecfb8ab113baef2/ignite/distributed/launcher.py#L176-L185
and there are also couple of `None` checks inside the function.

Reply:
Thanks, it makes sense. Could you please add `= None` to make `Optional` more explicit.